rasa/shared/core/training_data/story_reader/story_step_builder.py

```diff
@@ -74,31 +74,47 @@ def add_user_messages(
               Default value is `False`, which preserves the expected behavior
               of the reader.
         """
+        self.add_events(messages, is_used_for_training)
+
+    def add_events(
```

Comment:
Some of the comments preserved here might need to accommodate slot events, at the moment they only mention intents.